src/studio/src/designer/backend/Controllers/LanguagesController.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+using Altinn.Studio.Designer.Helpers;
+using Altinn.Studio.Designer.Services.Interfaces;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Studio.Designer.Controllers
+{
+    /// <summary>
+    /// Controller containing all actions related to languages and text-resources
```

Comment:
A bit to much responsibility in the comment here as text resources will be handled in the TextsController:)